ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -41,6 +41,41 @@ StackLayout {
 
     currentIndex:  chatsModel.activeChannelIndex > -1 && chatGroupsListViewCount > 0 ? 0 : 1
 
+    function populateSuggestions(){
+        chatInput.textInput.forceActiveFocus(Qt.MouseFocusReason)
```

Comment:
This shouldn't be in this function IMO. We don't want to force the focus on the input on new messages for example